UI/GameSettingsScreen.cpp

```diff
@@ -217,16 +217,25 @@ void GameSettingsScreen::CreateViews() {
 		return UI::EVENT_CONTINUE;
 	});
 	blockTransfer->SetDisabledPtr(&g_Config.bSoftwareRendering);
-	CheckBox *softwareGPU = graphicsSettings->Add(new CheckBox(&g_Config.bSoftwareRendering, gr->T("Software Rendering", "Software Rendering (slow)")));
-	softwareGPU->OnClick.Add([=](EventParams &e) {
-		if (g_Config.bSoftwareRendering)
-			settingInfo_->Show(gr->T("SoftGPU Tip", "Currently VERY slow"), e.v);
-		bloomHackEnable_ = !g_Config.bSoftwareRendering && (g_Config.iInternalResolution != 1);
-		return UI::EVENT_CONTINUE;
-	});
-	softwareGPU->OnClick.Handle(this, &GameSettingsScreen::OnSoftwareRendering);
-	if (PSP_IsInited())
-		softwareGPU->SetEnabled(false);
+
+	bool showSoftGPU = true;
+#if PPSSPP_PLATFORM(ANDROID)
```

Comment:
True.  But it was ANDROID before so still had that problem.

We probably should move to making some of the MOBILE_DEVICE checks runtime.  For example, it disables video recording, some debugger stuff, some reporting which might be slow, rewind I think, etc.  Many would be relevant on desktop/laptop/TV-ish Androids...

But at least it's in good company using MOBILE_DEVICE.  Worse is if we have to go hunting for 7 different defines if/when we decide to move some things to runtime...

-[Unknown]